packages/connector-chrome/package.json

```diff
@@ -14,17 +14,19 @@
   },
   "description": "hint connector for Google Chrome",
   "devDependencies": {
+    "@hint/parser-html": "^2.0.2",
```

Comment:
I'm not sure we should depend on the parser even if it's a `devDependency`. If the parser is the one that is going to do all the traversing then the related tests should be moved there and deleted from any connector.